docs/user/02-logs.md

```diff
@@ -20,9 +20,8 @@ The Telemetry module provides [Fluent Bit](https://fluentbit.io/) as a log agent
 2. Fluent Bit runs as a [DaemonSet](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/) (one instance per node), detects any new log files in the folder, and tails them using a filesystem buffer for reliability.
 3. Fluent Bit queries the [Kubernetes API Server](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/) for additional Pod metadata, such as Pod annotations and labels.
 4. The Telemetry module configures Fluent Bit with your custom output configuration.
-5. If Kyma's [deprecated](https://kyma-project.io/blog/2022/11/2/loki-deprecation/) logging component is installed, the Telemetry Manager configures the shipment to the in-cluster Loki instance automatically.
-6. As specified in your LogPipeline configuration, Fluent Bit sends the log data to observability systems outside or inside the Kyma cluster. Here, you can use the integration with HTTP to integrate a system directly or with an additional Fluentd installation.
-7. The user accesses the internal and external observability system to analyze and visualize the logs.
+5. As specified in your LogPipeline configuration, Fluent Bit sends the log data to observability systems outside or inside the Kyma cluster. Here, you can use the integration with HTTP to integrate a system directly or with an additional Fluentd installation.
+6. The user accesses the internal and external observability system to analyze and visualize the logs.
```

Comment:
Is "the user" someone else than the "you" we address in the previous step?

Reply:
```suggestion
6. To analyze and visualize your logs, access the internal and external observability system.
```

Reply:
wait, internal AND external? Or rather "or" ?

Reply:
The user can have only an internal observability system or only an external observability system or both. But I think "or" here fits better so I changed it.